routes/index.js

```diff
@@ -5,6 +5,8 @@ app.use('/healthcheck', require('./healthCheck.js'))
 app.use('/auth', require('./auth.js'))
 app.use('/users', require('./users.js'))
 app.use('/members', require('./members.js'))
+app.use('/tasks', require('./tasks'))
+app.use('/roadmap-site', require('./roadmap-site/index'))
```

Comment:
Please remove this in order to fix the failing checks
We're not using `roadmap-site` anymore